src/lib.rs

```diff
@@ -3,9 +3,10 @@
 // (see LICENSE or <http://opensource.org/licenses/MIT>) All files in the project carrying such
 // notice may not be copied, modified, or distributed except according to those terms.
 
-#![cfg_attr(feature = "doc", feature(external_doc))]
+#![cfg_attr(doc, feature(external_doc))]
```

Comment:
I have tried to find some info about this `cfg(doc)` but couldn't. Could you please point it out? Also IIRC, it gets enabled for doctests which makes our tests fail on stable. That was the main reason I made it a feature.